demo-ui/public/javascripts/ie.js

```diff
@@ -0,0 +1,494 @@
+/**
+* @preserve HTML5 Shiv 3.7.3 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
```

Comment:
is this not available on webjar? We shouldnt be copy pasting lib code into our project